### PR TITLE
Fixes #33814 - Job wizard add Purpose field

### PR DESCRIPTION
**Pull request by MariaAga**
Screenshot from 2021-11-02 14-14-33

### CODE REVIEW

File: webpack/JobWizard/steps/Schedule/index.js [hunk 1]
```diff
@@ -106,6 +108,16 @@ const Schedule = ({ scheduleValue, setScheduleValue, setValid }) => {
             }));
           }}
         />
+        <PurposeField
+          isDisabled={repeatType === repeatTypes.noRepeat}
```
Comment (laviro) [inline webpack/JobWizard/steps/Schedule/index.js:112]: should we hide the component when `repeatTypes === noRepeat` ?
it's pretty confusing when the field is disabled initially,
another option is to add more explanation on the tooltip

Comment (MariaAga) [reply webpack/JobWizard/steps/Schedule/index.js:112]: It only appears in `»Set up recurring execution«` in the original, so I don't think it has a use out of recurrence

Comment (adamruzicka) [reply webpack/JobWizard/steps/Schedule/index.js:112]: > so I don't think it has a use out of recurrence

Correct. Outside of recurring jobs it has no purpose, pun intended :)

**Comment by adamruzicka** [comment]
Also needs a rebase

**Comment by MariaAga** [comment]
thanks! rebased

**Review by adamruzicka — APPROVED** [review]
Looks good

**Comment by adamruzicka** [comment]
Thank you @MariaAga !

**Pull request merged**